PULL_REQUEST_MANAGEMENT.md

```diff
@@ -14,3 +14,9 @@ In addition, in order to avoid coming off as rude to helpful contributors, pleas
 - Asking the contributor to do unrelated work
 - Closing without providing a reason
 - Merging with the intention to rewrite that code soon after
+
+Before merging a PR, make sure it follows the [contributing rules](CONTRIBUTING.md#rules-and-recommendations), but more importantly:
+- Make sure the author does not use a nickname or alias, but a full legal name instead, and have specified a real e-mail in all PR commits
+- Make sure the author does have full name set in GitHub profile that matches one in commits
+- By pressing "Squash and merge" button in a PR you can make sure the author does not use no-reply e-mail -
```

Comment:
As discussed on an older PR of mine, real name and address used to be forwarded from PR commits to official repository.
Why not ask GitHub to fix current mess instead?

Also, what about even worse `committer GitHub <noreply@github.com>`?

Reply:
If all rules are met by the author (profile), there should be no problem regardless of the merge method used by a maintainer.

> Also, what about even worse `committer GitHub <noreply@github.com>`?

It's a corner case which is allowed.

Reply:
> Why not ask GitHub to fix current mess instead?

If GitHub decides to change interface with a feature to manually specify commit author's email, or select it from existing commits in the PR, then the rules could be updated.

Asking something always takes some time. We have limited manpower with limited time here.

Reply:
"We have limited manpower with limited time here, and despite that we must keep progressing."

Yes, and that is exactly the reason why I think we shouldn't make it too hard for external people to help us.

Reply:
I know there are some people who still helping us under an alias (e.g. I_Kill_Bugs), however they don't create PRs directly, instead maintainers themselves create commits based on their patches.

Reply:
> If all rules are met by the author (profile), there should be no problem regardless of the merge method used by a maintainer.

Which rules? A "no privacy" one? Select a process that rather does not require that...

> It's a corner case which is allowed, only for the committer of course.

Corner case? I would rather call it major, given the high number of affected commits.
Then, committer remains totally anonymous (as in that commit field is 100% useless), while author privacy is forbidden: NOT great!

> If GitHub decides to change interface with a feature to manually specify commit author's email, or select it from existing commits in the PR, then the rules could be updated.

Such feature behavior seems wanted, indeed.
In the meantime, that is nonsense: lost privacy could not be recovered afterward.

> Asking something always takes some time. It takes even more time to actually get things changed. We have limited manpower with limited time here, and despite that we must keep progressing.

So what? Did you even ask GitHub first? Why not just not use that feature if it doesn't work as needed (anymore/yet)?

Reply:
I have added a "Keep your privacy" point to the README.md.

Reply:
> Did you even ask GitHub first?

Yesterday, I eventually asked GitHub about this privacy with regard to multiple emails issue.
Hoping for an answer...

Reply:
> Hoping for an answer...

GitHub support response:
```
As you have discovered, if you have email privacy enabled, your noreply email address is the default commit author email address.
Unfortunately you cannot choose which email addresses to keep private and which email addresses to make public.
```